Dockerfile.tests

```diff
@@ -1,11 +1,11 @@
 # For running tests locally, see check_in_container target in Makefile
 
-FROM docker.io/usercont/packit
+FROM docker.io/usercont/packit-service-worker:dev
```

Comment:
We need a comment on how to get that image (I'd do 'docker-compose build', but how the others know?)